### PR TITLE
CODEOWNERS: Add me, @Ericson2314, to a few things please

### PR DESCRIPTION
**Pull request by Ericson2314**
###### Motivation for this change

Add myself two a few things. I'll line edit rationals.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -10,18 +10,19 @@
 # This file
 /.github/CODEOWNERS @edolstra
 
-# Boostraping and core infra
-/pkgs/stdenv                    @edolstra
-/pkgs/build-support/cc-wrapper  @edolstra
-
 # Libraries
 /lib                        @edolstra @nbp
+/lib/systems                @edolstra @nbp @ericson2314
```
Comment (Ericson2314) [inline .github/CODEOWNERS]: I overhauled this thing, a lot of stuff there is mainly needed for cross work / less obvious what it is used for outside of that. I'd also "nominate" @dezgeg to be an owner.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -10,18 +10,19 @@
 # This file
 /.github/CODEOWNERS @edolstra
 
-# Boostraping and core infra
```
Comment (Ericson2314) [inline .github/CODEOWNERS]: I made this category, but then "Nixpkgs Internals" was made, and I don't find this one sufficiently different from that one to warrant keeping it around.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -10,18 +10,19 @@
...
-/pkgs/top-level/default.nix @nbp
-/pkgs/top-level/impure.nix  @nbp
-/pkgs/top-level/stage.nix   @nbp
+/default.nix                          @nbp
+/pkgs/top-level/default.nix           @nbp @Ericson2314
+/pkgs/top-level/impure.nix            @nbp @Ericson2314
+/pkgs/top-level/stage.nix             @nbp @Ericson2314
+/pkgs/stdenv                          @edolstra
+/pkgs/build-support/cc-wrapper        @edolstra @Ericson2314
+/pkgs/build-support/bintools-wrapper  @edolstra @Ericson2314
```
Comment (Ericson2314) [inline .github/CODEOWNERS]: The PR of mine making this was just merged, and `binutils-wrapper` just as important as `cc-wrapper` (from which it was factored out of) and so everyone owning `cc-wrapper` should own it too.

**Comment by grahamc** [comment]
@GrahamcOfBorg eval

**Review by nbp — APPROVED** [review]

**Review by globin — APPROVED** [review]

**Comment by Ericson2314** [comment]
Thanks!

**Pull request merged**